packages/bull-vault/src/LeverageBull.sol

```diff
@@ -125,16 +125,16 @@ contract LeverageBull is Ownable {
         uint256 _squeethInCrab,
         uint256 _crabTotalSupply
     ) internal returns (uint256, uint256, uint256) {
-        (uint256 ethToLend, uint256 usdcToBorrow) = _calcLeverageEthUsdc(
+        (uint256 wethToLend, uint256 usdcToBorrow) = _calcLeverageEthUsdc(
             _crabAmount, _bullShare, _ethInCrab, _squeethInCrab, _crabTotalSupply
         );
 
-        require(ethToLend == _ethAmount, "LB0");
+        require(wethToLend == _ethAmount, "LB0");
 
-        _depositEthInEuler(ethToLend, true);
+        _depositEthInEuler(wethToLend, true);
```

Comment:
`_depositEthInEuler` => `_depositWethInEuler`

Reply:
happy to change this one as well